src/ai/backend/agent/docker/agent.py

```diff
@@ -1240,7 +1240,6 @@ class DockerAgent(AbstractAgent[DockerKernel, DockerKernelCreationContext]):
     monitor_docker_task: asyncio.Task
     agent_sockpath: Path
     agent_sock_task: asyncio.Task
-    scan_images_timer: asyncio.Task
```

Comment:
The timer is created in the line below, but this variable is not used.

https://github.com/lablup/backend.ai/blob/1f77864bd706111fc55d6f3733fdf58439b0c446/src/ai/backend/agent/agent.py#L755